### ISSUE TITLE
Alternative version without cloth config included?

### ISSUE THREAD

**Issue by RDKRACZ** [juliand665/Dynamic-FPS#41]
Most people already have cloth config like 30 times because of other mods. Can you release an additional version without it?
Maybe you could switch cloth for some other config system ex. [Omega config](https://github.com/Draylar/omega-config)?

**Comment by juliand665** [comment]
I've been weighing the pros and cons of requiring cloth as an external dependency rather than embedding it. I suppose the mod already requires Fabric API, and cloth tends to be the main thing that breaks with updates while the mod itself just keeps working. #29 recently addressed this but was closed by the author in favor of Quilt's upcoming built-in config system.

**Comment by juliand665** [comment]
Yeah, that's a great idea. I'm honestly not sure why I didn't do it sooner—I think I was scared it might break regular config functionality, but it doesn't seem to. I think I will indeed not be jar-in-jaring cloth config anymore—here's a pre-release of sorts without it, lol (still can't believe GitHub doesn't let you upload raw JAR files).
[dynamic-fps-2.0.6.jar.zip](https://github.com/juliand665/Dynamic-FPS/files/7358365/dynamic-fps-2.0.6.jar.zip)

**Comment by Madis0** [comment]
Why not release it for everyone then?

**Comment by altrisi** [comment]
I'd assume it's here so the issue authors (and anyone interested) can test it first, and then the release will come, just like in previous issues.
